fdbserver/VersionedBTree.actor.cpp

```diff
@@ -2959,7 +2991,7 @@ class DWALPager : public IPager2 {
 		uint32_t queueCount;
 		uint32_t pageSize;
 		int64_t pageCount;
-		uint32_t pagesPerExtent;
+		uint32_t extentSize;
```

Comment:
This isn't quite what I meant - Internally, the extent size must be a multiple of the page size, so internally it should still be stored as a page count.  But for user-facing configuration, such as the configuration knob (or later, storage engine parameters at the cluster level) or for trace log details it is better for the user to deal with bytes.  

So the idea is to accept an extent size in bytes, but then convert it internally to a whole page count (integer division with an implicit round down is fine), so you can roll back the internal changes and just have the constructor make the conversion.

Reply:
I do convert the extent size into a page count and that's stored as part of pager (see setExtentSize()). I just changed what we store in pager header as the user specified byte size. Rest of the code deals directly with pagesPerExtent.

But I agree that doing the conversion in the constructor is another way.

Reply:
I see, I like what you've done.  Storing the user-chosen size in the header makes sense!